MobileBuy/sample/src/main/java/com/shopify/sample/activity/CheckoutActivity.java

```diff
@@ -53,7 +54,15 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setTitle(R.string.checkout);
-        setContentView(R.layout.checkout_activity);
+
+        OptimizelyClient optimizelyClient = getSampleApplication().getOptimizelyManager().getOptimizely();
+        Variation product_sorting_variation = optimizelyClient.getVariation("checkout_activity_layout",
+                getSampleApplication().getUser());
+        if (product_sorting_variation.getKey().equals("include_fake_text")) {
+            setContentView(R.layout.checkout_activity);
```

Comment:
why not just change the button text instead of changing the whole layout? The only difference between the two layout files it he text for the button.